examples/node-no-lockfile-canvas/index.js

```diff
@@ -0,0 +1,2 @@
+const canvas = require("canvas")
```

Comment:
```suggestion
require("canvas");
```

Setting it to a variable seems unnecessary as it isn't used.